### ISSUE TITLE
Issues with certain applications

### ISSUE THREAD

**Issue by Mrsevic** [atonamy/SignalingServer#1]
Hi!

I've tried your server with certain simple WebRtc JS applications and I get the errors that I am passing null values.

Do you know about any sample WebRtc application on the internet which may be able to utilize this piece of code?

Thanks

**Comment by atonamy** [comment]
@Mrsevic  Yes this is sample webrtc application https://github.com/atonamy/JustCallMe which is working together with this SignalingServer . You can see in source code how to use it.

**Comment by Mrsevic** [comment]
I see. Thanks for this. Do you by any chance know some web node.js clients that might be able to utilize this server?

**Comment by atonamy** [comment]
@Mrsevic I think any node.js client as long as it support websocket. The only thing you have to do your own implementation to communicate with this server (request/response threads), there no standard client for it.

You can even use pure javascript for WebRtc messaging using this server.
